index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+    integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <nav class='navbar navbar-expand-lg navbar-light justify-content-between bg-light'>
+        <a class="nav-link" href="#about">About</a>
+        <a class="nav-link" href="#achievements">Achievements</a>
+        <a class="nav-link" href="#Gallery">Gallery</a>
+    </nav>
+    <div class="container">
+        <h1>Jean-Claude Van Damme</h1>
+        
+    </div>
+    <div >
+        <section id="About" class="container-fluid">
+            <div class="row align-items-center content">
```

Comment:
Its a bettter practice to use class than id since classes can be assigned to multiple elements on a page or throughout the your website